integration/ct_integration_test.sh

```diff
@@ -3,6 +3,10 @@ set -e
 INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
 . "${INTEGRATION_DIR}"/common.sh
 
+yes | "${SCRIPTS_DIR}"/resetdb.sh
```

Comment:
Yes, it's not very subtle :)

I think we can make it work just with wipelog, so let me know if you'd like it.